readthedocs/builds/models.py

```diff
@@ -732,6 +732,14 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('builds_detail', args=[self.project.slug, self.pk])
 
+    def get_full_url(self):
+        """Get full url including domain"""
+        full_url = '{domain}{absolute_url}'.format(
+            domain=settings.PUBLIC_API_URL,
```

Comment:
This isn't the right setting. 
```suggestion
            domain=settings.PRODUCTION_DOMAIN,
```

Reply:
`PRODUCTION_DOMAIN` does not include `http` / `https` schemes.

Reply:
Do you have any suggestions on which constant should I use?

Reply:
Looks like we already just assume `https` in the code: https://github.com/readthedocs/readthedocs.org/blob/f886e68260d1dd37135fb20278b40502f9b7d7b0/readthedocs/oauth/services/github.py#L172-L182

Reply:
@ericholscher we wont be able to see the page when using `localhost` for testing as `localhost` does not support `https`